### ISSUE TITLE
InvoiceTaxAmount is Missing "taxable_amount" Mapped Property Support

### ISSUE THREAD

**Issue by nox7** [stripe/stripe-dotnet#2918]
### Describe the bug

The InvoiceTaxAmount object should have a TaxableAmount property mapped to the JSON property "taxable_amount." It is not present.

### To Reproduce

1. Fetch an Invoice
2. Read the "total_tax_amounts" (TotalTaxAmounts property)
3. No TaxableAmount 

### Expected behavior

A mapped property TaxableAmount to the JSON property "taxable_amount"

### OS

Windows

### .NET version

.NET 8

### Library version

stripe-dotnet v44.9.0

### API version

2024-04-10

**Comment by remi-stripe** [comment]
@nox7 I'm not really following up what your ask is about. Our Invoice API has the `total_tax_amounts` [property](https://docs.stripe.com/api/invoices/object#invoice_object-total_tax_amounts) already which is an array of objects with more details about each one. This has been supported by stripe-dotnet for a long time and is defined in our types [here](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Entities/Invoices/Invoice.cs#L969) like this
```
[JsonProperty("total_tax_amounts")]
public List<InvoiceTaxAmount> TotalTaxAmounts { get; set; }
```

**Comment by nox7** [comment]
@remi-stripe That isn't the property I am referring to.

Please see this object: https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Entities/Invoices/InvoiceTaxAmount.cs

It is missing the below mapped property from the Stripe API

![image](https://github.com/stripe/stripe-dotnet/assets/17110935/76ccd532-0eb1-4abf-be63-451a22a1fd66)

**Comment by remi-stripe** [comment]
Reopening as this is not fixed yet. We will release a new version tomorrow with the fix!

**Comment by anniel-stripe** [comment]
The fix was released in version v44.10.0 today.
